### PR TITLE
Fixing refresh token

### PR DESCRIPTION
**Pull request by jwhughes93**

### CODE REVIEW

File: networking-core/src/networking.js [hunk 1]
```diff
@@ -20,6 +22,8 @@ export class Networking {
         const tokenManager = Networking.tm;
         const accessToken = useAnonToken ? this.config.defaultToken : await tokenManager.getAccessToken();
 
+        DEBUG && DEBUG && console.log( 'Networking:getHeaders', 'accessToken', { accessToken } );
```
Comment (opetroulas) [inline networking-core/src/networking.js]: DEBUG && DEBUG????

**Comment by ksmoro** [comment]
networking-core shouldn't link to UIEngine in networking.js 
Maybe make the ENUMS in networking-core to remove the cross dependency.

File: networking-core/src/networking.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import {STATUS_CODES} from "./consts";
+import {ENUMS} from "../../uiengine";
```
Comment (ksmoro) [inline networking-core/src/networking.js]: Make the specific ENUMS local to networking.js so that networking isn't reliant on importing uiengine

Comment (jwhughes93) [reply networking-core/src/networking.js]: good call

**Pull request merged**